t/005-apicast-oauth.t

```diff
@@ -791,6 +805,86 @@ GET /t
 Content-Type: application/json
 --- error_code: 200
 --- response_body
+{"token_type":"bearer","expires_in":123,"access_token":"token"}
+--- no_error_log
+[error]
+
+=== TEST 18: default token TTL
+When a default TTL is not specified, it applies a default of 7 days (604800 s)
```

Comment:
Can you try with empty value? That is common OpenShift fuckup.

Reply:
Good point. I added a new test (`TEST 19`) which is basically a copy-pasta of `TEST 18`. The only difference is an env. Any ideas to avoid this kind of duplication in Test::Nginx ?

Reply:
No idea :( It really sucks to have this kind of duplication.

Reply:
Maybe would be useful to test it on lower level - in busted.

Reply:
Agreed. I thought about that, but then I saw that there are _0_ unit tests for oauth, so I thought that it would be better to just complete the existing integration tests for now. Maybe this is something we could address in a future PR.

Reply:
Definitely. 👍